README.md

```diff
@@ -11,7 +11,7 @@ The OBS consists of a backend and a frontend. The backend implements all the cor
 The Open Build Service is Free Software and is released under the terms of the GPL, except where noted. Additionally, 3rd-party content (like, but not exclusively, the webui icon theme) may be released under a different license. Please check the respective files for details.
 
 ## Community
-You can discuss with the OBS Team via IRC on the channel [#opensuse-buildservice](irc://freenode.net/opensuse-buildservice). Or you can use our mailing list [opensuse-buildservice@opensuse.org](mailto:opensuse-buildservice+subscribe@opensuse.org).
+You can discuss with the OBS Team via IRC on the channel [#opensuse-buildservice](irc://freenode.net/opensuse-buildservice). Or you can use our mailing list [opensuse-buildservice@opensuse.org](mailto:opensuse-buildservice+subscribe@opensuse.org). Please refer to the openSUSE Mailing Lists [page](https://en.opensuse.org/openSUSE:Mailing_lists_subscription#Subscribing) to learn about our mailing lists (un)subscription and more.
```

Comment:
what do you mean with more? :thinking:

Reply:
What to write, how to write it, how to format it. Everything that is on that page. Have you read it? 🤠

Reply:
What do you guys think of '... to learn about subscription and additional information.'? Let's not mention unsubscription, it's bad karma.